### ISSUE TITLE
Build fails on Windows 10 (for another reason)

### ISSUE THREAD

**Issue by stelioskat** [eddex/aseprite-windows-docker-build#15]
This is the whole output when I run `start.bat`.  I really can't understand why it cannot find the image, since the image is actually still available.

```
>start.bat
Building Aseprite using Docker
[+] Building 436.5s (5/7)                                                                                                                                                                                                     docker:default
 => [internal] load .dockerignore                                                                                                                                                                                                       0.0s
 => => transferring context: 2B                                                                                                                                                                                                         0.0s
 => [internal] load build definition from Dockerfile                                                                                                                                                                                    0.0s
 => => transferring dockerfile: 117B                                                                                                                                                                                                    0.0s
 => [internal] load metadata for docker.io/abrarov/msvc-2022:2.15.0                                                                                                                                                                     1.8s
 => [internal] load build context                                                                                                                                                                                                       0.0s
 => => transferring context: 903B                                                                                                                                                                                                       0.0s
 => ERROR [1/3] FROM docker.io/abrarov/msvc-2022:2.15.0@sha256:108339140b0279672903b7c10d74116d89164dd5ac657f79bb40135dc44b83ed                                                                                                       434.5s
 => => resolve docker.io/abrarov/msvc-2022:2.15.0@sha256:108339140b0279672903b7c10d74116d89164dd5ac657f79bb40135dc44b83ed                                                                                                               0.0s
 => => sha256:b111c3320c949bea81612bf4554f1b6592c2f504920b5bf57ba340a1d4d52c93 1.88GB / 1.88GB                                                                                                                                        334.4s
 => => sha256:ca73064362226b7df47e746a33e9d3f10628ad09224721ff795618c85e2ff104 1.29kB / 1.29kB                                                                                                                                          0.3s
 => => sha256:f49435d194108cd56f173ad5bc6a27c70eed98b7e8cd54488f5acd85efbd51c9 7.69kB / 7.69kB                                                                                                                                          0.0s
 => => sha256:108339140b0279672903b7c10d74116d89164dd5ac657f79bb40135dc44b83ed 3.61kB / 3.61kB                                                                                                                                          0.0s
 => => sha256:294d77ac553d7210b662d07674ef9e39a6c2e88dc15b9d2788d51e509bc8b54e 800.55MB / 800.55MB                                                                                                                                    239.9s
 => => sha256:52e8bf960487dc434a52fd2c35af27d90be2a68350e23f11e4e3e8efe5f1e74a 3.81kB / 3.81kB                                                                                                                                          0.5s
 => => sha256:d0423104b1e8eb4ba8f6f8b92f29f150f61c99e3ae0155329524ac2b435ae82f 1.26GB / 1.26GB                                                                                                                                         88.0s
 => => sha256:e955cb691f75ed421e352c2b78022deb5a85b9b9ecf344a3f8ed9e3da1f896aa 1.29kB / 1.29kB                                                                                                                                         88.7s
 => => sha256:12ba4d046135d63a754b968f6937a1fd6222c1da6862395b9e85e3e2174c2af5 1.29kB / 1.29kB                                                                                                                                         88.8s
 => => sha256:ab9c362daf8c54ee2d2df390e3fdba4380942f37abe0c599fd5816a565af61b1 1.29kB / 1.29kB                                                                                                                                         89.1s
 => => sha256:51697aeca9c923aafcaf27c9bb41acce16613190efcd28e4c5c0bd7d94837e2a 1.28kB / 1.28kB                                                                                                                                         89.2s
 => => sha256:f70742e625f6f5679c93c40c2f866804438ed81c58b7f5c3e90dbb2e7dc311b6 2.44kB / 2.44kB                                                                                                                                         89.5s
 => => sha256:d8ad052ff44426c3b9ed8e60880065e2bfe232a1cce8109b8fc5521150f1d942 1.93GB / 1.93GB                                                                                                                                        207.2s
 => => sha256:38911ee4e4541823cdf83482d30d85ac7b786dc0ef7e0ee4495b47225360bcae 1.33kB / 1.33kB                                                                                                                                        207.6s
 => => sha256:a9fbf00bb9e2c596caa789fa21bc99b3ba4f80f921f3e569ac16019d5b01c3c9 1.29kB / 1.29kB                                                                                                                                        207.9s
 => => sha256:05e5f3eaa5fee2508b8b10b748fc39800dba467e43617d4e090b9c8d31ced6f1 1.29kB / 1.29kB                                                                                                                                        208.2s
 => => extracting sha256:b111c3320c949bea81612bf4554f1b6592c2f504920b5bf57ba340a1d4d52c93                                                                                                                                              86.0s
 => => extracting sha256:294d77ac553d7210b662d07674ef9e39a6c2e88dc15b9d2788d51e509bc8b54e                                                                                                                                              13.5s
------
 > [1/3] FROM docker.io/abrarov/msvc-2022:2.15.0@sha256:108339140b0279672903b7c10d74116d89164dd5ac657f79bb40135dc44b83ed:
------
Dockerfile:3
--------------------
   1 |     # escape=`
   2 |
   3 | >>> FROM abrarov/msvc-2022:2.15.0
   4 |
   5 |     COPY build.bat C:\
--------------------
ERROR: failed to solve: failed to register layer: link /Files/Program Files/common files/Microsoft Shared/Ink/en-US/micaut.dll.mui /Files/Program Files (x86)/common files/Microsoft Shared/ink/en-US/micaut.dll.mui: no such file or directory
Copy Aseprite from Docker container
Unable to find image 'eddex/aseprite:latest' locally
docker: Error response from daemon: pull access denied for eddex/aseprite, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
Error response from daemon: No such container: aseprite
Error response from daemon: No such container: aseprite
Error response from daemon: No such container: aseprite
Error response from daemon: No such image: eddex/aseprite:latest
Done
```



If I try to pull this image alone I get the following strange error:
```
>docker pull docker.io/abrarov/msvc-2022:2.15.0
2.15.0: Pulling from abrarov/msvc-2022
b111c3320c94: Pulling fs layer
294d77ac553d: Pulling fs layer
ca7306436222: Download complete
52e8bf960487: Waiting
...
image operating system "windows" cannot be used on this platform: operating system is not supported
```

**Comment by stelioskat** [comment]
Turns out I just needed to change to use "Windows containers" instead of Linux. 

This was also a bit tricky as I had to enable a Windows feature through the powershell, which also did not work at first. After lots of Googling, what worked was:

1. Open the Powershell and run `Enable-WindowsOptionalFeature -Online -FeatureName $("Microsoft-Hyper-V", "Containers") -All`
2. Reboot
3. Install Docker Desktop **v4.21.1**
4. Reboot
5. Right-click on the Docker tray icon and select "Switch to Windows containers..." 
6. Reboot
7. Then successfully run `start.bat`.
